packages/app/tests/e2e/features/artifacts/artifactsSet1.feature

```diff
@@ -4,53 +4,30 @@ Feature: Main Page
   Background:
     Given I am on main page
 
-  @id253 @featureEnv @testnet
-  Scenario Outline: Check the element "<Sub-Section>" in Tools section is available, clickable and have correct href
-    Given I click by text "Tools"
-    Given Element with "text" "<Sub-Section>" should be "visible"
-    When Element with "text" "<Sub-Section>" should be "clickable"
-    Then Element with "text" "<Sub-Section>" should have "<url>" value
-
-    Examples:
-      | Sub-Section                 | url                                       |
-      | Smart Contract Verification | /contracts/verify                         |
-      #| Portal                      | https://goerli.staging-portal.zksync.dev/ |
-
-  @id253 @featureEnv @mainnet
-  Scenario Outline: Check the element "<Sub-Section>" in Tools section is available, clickable and have correct href
-    Given I click by text "Tools"
-    Given Element with "text" "<Sub-Section>" should be "visible"
-    When Element with "text" "<Sub-Section>" should be "clickable"
-    Then Element with "text" "<Sub-Section>" should have "<url>" value
 
-    Examples:
-      | Sub-Section                 | url                                       |
-      | Smart Contract Verification | /contracts/verify                         |
-      #| Portal                      | https://staging-portal.zksync.dev/        |
-
-  @id253:I @productionEnv @testnet
-  Scenario Outline: Check the element "<Sub-Section>" in Tools section is available, clickable and have correct href
+  @id253:I @featureEnv @testnetSmokeSuite @testnet
+  Scenario Outline: Check the element "<Sub-Section>" in Tools section is available, clickable and have correct href (Sepolia)
     Given I click by text "Tools"
     Given Element with "text" "<Sub-Section>" should be "visible"
     When Element with "text" "<Sub-Section>" should be "clickable"
     Then Element with "text" "<Sub-Section>" should have "<url>" value
 
     Examples:
-      | Sub-Section                 | url                                       |
-      | Smart Contract Verification | /contracts/verify                         |
-      #| Portal                      | https://goerli.portal.zksync.io/          |
+      | Sub-Section                 | url                                              |
+      | Smart Contract Verification | /contracts/verify                                |
+      | Bridge                      | https://portal.zksync.io/bridge/?network=sepolia |
 
-  @id253:I @productionEnv @mainnet
-  Scenario Outline: Check the element "<Sub-Section>" in Tools section is available, clickable and have correct href
+  @id253:II @featureEnv @mainnet
```

Comment:
@olehbairak  If it works on mainnet in @productionEnv as well as in @featureEnv then just remove this tag to be able run it in both places (it is just for mainnet)

Reply:
Fixed already. Thank you for the feedback!